### DISCUSSION TITLE
Ontvangt de RF plugin ook de huidige status?

### DISCUSSION THREAD

**Discussion by BasvanH** [arnemauer/Ducobox-ESPEasy-Plugin#126] [Q&A] [answered]
Ik ben benieuwd of de RF plugin ook de huidige status van de ventilator ontvangt. Dus als ik via de bedrade of andere RF afstandsbediening de stand wijzig, of de RF plugin dit dan ook registreert en kan doorsturen.

Ik lees de zin "De unit gedraagt zich als een CO2 bedieningsschakelaar. De unit registreert de actuele ventilatiemodus, kan de ventilatiemodus wijzigen en kan een override percentage instellen (ventilatiesnelheid ongeacht de gekozen modus)."

Maar wordt met **unit** nu aan de RF plugin of de ventilator gerefereerd?

**Comment by arnemauer** [comment, marked as answer]
Met unit wordt de rf plugin bedoeld. De ducobox stuurt naar alle bedieningsschakelaars de actuele stand, dus ook naar de rf plugin. De seriële plugin kan ook de ventilatiestand uitlezen maar dat heeft niet de voorkeur omdat de seriële plugin elke 60 seconden de data uitleest. Er zit dan een vertraging in.

**Comment by BasvanH** [comment]
@arnemauer, ok dank je wel.
